templates/epub2/content.opf.ejs

```diff
@@ -13,7 +13,7 @@
         <dc:creator opf:role="aut" opf:file-as="<%= author.length ? author.join(",") : author %>"><%= author.length ? author.join(",") : author %></dc:creator>
         <dc:date opf:event="modification"><%= date %></dc:date>
         <dc:language><%= lang || "en" %></dc:language>
-        <meta name="cover" content="image_cover" />
+        <% if(locals.cover || (locals.firstImageIsCover && images.length)) { %><meta name="cover" content="<%= locals.cover ? 'image_cover' : 'image_0' %>"/><% } %>
```

Comment:
I wonder if we can avoid these `if` by puting some logic in this method https://github.com/lesjoursfr/html-to-epub/blob/6e68bcde98b269659b14fab96440ace99223ad0e/src/index.ts#L693